tachyon/math/base/groups.h

```diff
@@ -48,7 +48,7 @@ class MultiplicativeGroup : public MultiplicativeSemigroup<G> {
   template <
       typename G2,
       std::enable_if_t<internal::SupportsDivInPlace<G, G2>::value>* = nullptr>
-  constexpr auto& operator/=(const G2& other) {
```

Comment:
What other types could come if we haven't forced to return `G&`?

Reply:
Only `G&` can come. 

```c++
T a;
a += b; // a doesn't change its type!
```